src/viam/components/power_sensor/power_sensor.py

```diff
@@ -57,6 +57,7 @@ async def get_readings(self, *, extra: Optional[Dict[str, Any]] = None, timeout:
                is_ac: bool
                power: float
             }
+
```

Comment:
should the below be removed? in `sensor.py` the only thing in `get_readings` is ...